items/item-history.js

```diff
@@ -408,8 +409,12 @@ class ItemHistory {
     try {
       quantityState = Quantity(rawState.toString());
     } catch (e) {
-      if (e instanceof QuantityError) quantityState = null;
-      throw Error('Failed to create "quantityState": ' + e);
+      if (e instanceof QuantityError) {
+        log.debug(`Failed to create "quantityState" for Item ${this.name}:`, e);
```

Comment:
Do we want to leave debug messages like this here?  I wonder if this is just adding noise?

Reply:
I agree, I will remove this logging.